src/sentry/south_migrations/0500_auto__chg_field_group_project.py

```diff
@@ -0,0 +1,1467 @@
+# -*- coding: utf-8 -*-
+from south.utils import datetime_utils as datetime
+from south.db import db
+from south.v2 import SchemaMigration
+from django.db import models
+
+
+class Migration(SchemaMigration):
+
+    # Flag to indicate if this migration is too risky
+    # to run online and needs to be coordinated for offline
+    is_dangerous = False
```

Comment:
We absolutely need this `is_dangerous = True` so we don't attempt to run this in production.

Reply:
ok, done